dev/AnimatedVisualPlayer/AnimatedVisualPlayer.idl

```diff
@@ -62,6 +80,9 @@ unsealed runtimeclass AnimatedVisualPlayer
     [MUX_DEFAULT_VALUE("true")]
     [MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
     Boolean AutoPlay;
+    [MUX_DEFAULT_VALUE("true")]
+    [MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
+    Boolean Active;
```

Comment:
This will need to be marked as Preview as well.

Reply:
Not sure what the syntax should be, I tried just `[MUX_PREVIEW]` or `[MUX_PREVIEW] { ... }` but it resulted in runtime error:
![image](https://user-images.githubusercontent.com/83784664/139968448-bd7eef76-5ae8-4f00-a84a-4e87bee07a73.png)